test/test_madx.py

```diff
@@ -102,10 +102,14 @@ def test_streamreader(self):
         output = []
         m = Madx(stdout=output.append)
         self.assertEqual(len(output), 1)
-        self.assertIn(b'++++++++++++++++++++++++++++++++++++++++++++', output[0])
```

Comment:
was there a particular reason for the new spelling (I find the old one more clearly laid out)

Reply:
my fault, I have used autopep8 on the file